### PR TITLE
punched up tagline

### PR DESCRIPTION
**Pull request by dju11**
Hey guys, I changed the tagline. Hope you like it!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <div class="wrapper">
       <header>
         <h1>Berkshire County Technology Group</h1>
-        <p>This is a group for people in Berkshire County, MA interested in engaging with all aspects of technology.</p>
+        <p>A group for people interested in all aspects of technology</p>
```
Comment (tristil) [inline index.html:19]: Are you sure about A? How about 'an'?

Comment (dju11) [inline index.html:19]: No, dude, that's for words that start with vowels. Duh!

Comment (tristil) [inline index.html:19]: Oh yeah. nm :+1:

**Comment by tristil** [comment]
:cake:

**Pull request merged**